<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserItem</class>
 <widget class="QWidget" name="UserItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="avatarLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="onOrOffLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="unreadLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(255, 0, 0);
height:20px;
color: rgb(255, 255, 255);
padding:2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
